build.ts

```diff
@@ -183,7 +185,8 @@ db.exec(`
    field_area INTEGER,
    spawns_with_lotm BOOL,
    korok_id TEXT,
-   korok_type TEXT
+   korok_type TEXT,
+   location TEXT
```

Comment:
Are there any plans to add support for the other LocationTags? If not this should probably be renamed to make it clear this column is only used for Hyrule Castle areas

Reply:
Other possible locations where this might be useful are:
- Akkala Labyrinth Underground
- Yiga hideout
- Forgotten Temple
- Hebra Great Skeleton
- near Dow Na'eh Shrine (behind a waterfall)
- there are probably others

All of these are locations are underground